How to use boost::corotine2?

**Describe the bug**
I can not use the boost coroutine2 package. It was not found.

**Environment**
- OS: macOS 11 on Apple M1
- Compiler: Apple clang version 12.0.0 (clang-1200.0.32.29)

**To Reproduce**
Steps to reproduce the behavior:
1. ./vcpkg install boost-coroutine2

In a cmake project:
```cmake
find_package(Boost COMPONENTS coroutine2 REQUIRED)
```
does not work, see error below.
But finding other packages works. So the following lines does not produce any errors:
```cmake
find_package(Boost COMPONENTS context thread REQUIRED)
target_link_libraries(lichtsteuerung Boost::boost Boost::context Boost::thread)
```

**Expected behavior**
find_package finds package `coroutine2`

**Failure logs** 
<details>
<summary>Error log</summary>
<pre>
Running /usr/local/Cellar/cmake/3.19.4/bin/cmake -S /Users/xxx/git_projekte/Lichtsteuerung -B /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug in /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug.
-- Running vcpkg install
Detecting compiler hash for triplet x64-osx...
All requested packages are currently installed.

Total elapsed time: 1.458 us

-- Running vcpkg install - done
CMake Warning at /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1204 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1326 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1935 (_Boost_MISSING_DEPENDENCIES)
  vcpkg/scripts/buildsystems/vcpkg.cmake:614 (_find_package)
  src/CMakeLists.txt:136 (find_package)


CMake Warning at /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1204 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1326 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1935 (_Boost_MISSING_DEPENDENCIES)
  vcpkg/scripts/buildsystems/vcpkg.cmake:614 (_find_package)
  src/CMakeLists.txt:136 (find_package)


CMake Warning at /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1204 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1326 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1935 (_Boost_MISSING_DEPENDENCIES)
  vcpkg/scripts/buildsystems/vcpkg.cmake:614 (_find_package)
  src/CMakeLists.txt:136 (find_package)


CMake Warning at /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1204 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1326 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1935 (_Boost_MISSING_DEPENDENCIES)
  vcpkg/scripts/buildsystems/vcpkg.cmake:614 (_find_package)
  src/CMakeLists.txt:136 (find_package)


CMake Warning at /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1204 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1326 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1935 (_Boost_MISSING_DEPENDENCIES)
  vcpkg/scripts/buildsystems/vcpkg.cmake:614 (_find_package)
  src/CMakeLists.txt:136 (find_package)


CMake Warning at /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1204 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1326 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:1935 (_Boost_MISSING_DEPENDENCIES)
  vcpkg/scripts/buildsystems/vcpkg.cmake:614 (_find_package)
  src/CMakeLists.txt:136 (find_package)


CMake Error at /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:218 (message):
  Could NOT find Boost (missing: coroutine) (found version "1.75.0")
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:582 (_FPHSA_FAILURE_MESSAGE)
  /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/FindBoost.cmake:2193 (find_package_handle_standard_args)
  vcpkg/scripts/buildsystems/vcpkg.cmake:614 (_find_package)
  src/CMakeLists.txt:136 (find_package)


-- Configuring incomplete, errors occurred!
See also "/Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeOutput.log".
CMake process exited with exit code 1.
Elapsed time: 00:04.
</pre>
</details>
<details><summary>"CMakeOutput.log"</summary>
<pre>
The system is: Darwin - 20.2.0 - x86_64
Compiling the CXX compiler identification source file "CMakeCXXCompilerId.cpp" succeeded.
Compiler: /usr/bin/clang++ 
Build flags: 
Id flags:  

The output was:
0


Compilation of the CXX compiler identification source "CMakeCXXCompilerId.cpp" produced "a.out"

The CXX compiler identification is AppleClang, found in "/Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/3.19.4/CompilerIdCXX/a.out"

Detecting CXX compiler ABI info compiled with the following output:
Change Dir: /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeTmp

Run Build Command(s):/usr/bin/make cmTC_fe95f/fast && /Applications/Xcode.app/Contents/Developer/usr/bin/make  -f CMakeFiles/cmTC_fe95f.dir/build.make CMakeFiles/cmTC_fe95f.dir/build
Building CXX object CMakeFiles/cmTC_fe95f.dir/CMakeCXXCompilerABI.cpp.o
/usr/bin/clang++   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk   -v -Wl,-v -o CMakeFiles/cmTC_fe95f.dir/CMakeCXXCompilerABI.cpp.o -c /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/CMakeCXXCompilerABI.cpp
Apple clang version 12.0.0 (clang-1200.0.32.29)
Target: x86_64-apple-darwin20.2.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
clang: warning: -Wl,-v: 'linker' input unused [-Wunused-command-line-argument]
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include/c++/v1"
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang" -cc1 -triple x86_64-apple-macosx11.0.0 -Wdeprecated-objc-isa-usage -Werror=deprecated-objc-isa-usage -Werror=implicit-function-declaration -emit-obj -mrelax-all -disable-free -disable-llvm-verifier -discard-value-names -main-file-name CMakeCXXCompilerABI.cpp -mrelocation-model pic -pic-level 2 -mthread-model posix -mframe-pointer=all -fno-strict-return -masm-verbose -munwind-tables -target-sdk-version=11.1 -target-cpu penryn -dwarf-column-info -debugger-tuning=lldb -target-linker-version 609.8 -v -resource-dir /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -stdlib=libc++ -internal-isystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1 -internal-isystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/local/include -internal-isystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include -internal-externc-isystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include -internal-externc-isystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include -Wno-reorder-init-list -Wno-implicit-int-float-conversion -Wno-c99-designator -Wno-final-dtor-non-final-class -Wno-extra-semi-stmt -Wno-misleading-indentation -Wno-quoted-include-in-framework-header -Wno-implicit-fallthrough -Wno-enum-enum-conversion -Wno-enum-float-conversion -fdeprecated-macro -fdebug-compilation-dir /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeTmp -ferror-limit 19 -fmessage-length 0 -stack-protector 1 -fstack-check -mdarwin-stkchk-strong-link -fblocks -fencode-extended-block-signature -fregister-global-dtors-with-atexit -fgnuc-version=4.2.1 -fobjc-runtime=macosx-11.0.0 -fcxx-exceptions -fexceptions -fmax-type-align=16 -fdiagnostics-show-option -o CMakeFiles/cmTC_fe95f.dir/CMakeCXXCompilerABI.cpp.o -x c++ /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/CMakeCXXCompilerABI.cpp
clang -cc1 version 12.0.0 (clang-1200.0.32.29) default target x86_64-apple-darwin20.2.0
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/local/include"
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/Library/Frameworks"
#include "..." search starts here:
#include <...> search starts here:
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks (framework directory)
End of search list.
Linking CXX executable cmTC_fe95f
/usr/local/Cellar/cmake/3.19.4/bin/cmake -E cmake_link_script CMakeFiles/cmTC_fe95f.dir/link.txt --verbose=1
/usr/bin/clang++  -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  -v -Wl,-v CMakeFiles/cmTC_fe95f.dir/CMakeCXXCompilerABI.cpp.o -o cmTC_fe95f 
Apple clang version 12.0.0 (clang-1200.0.32.29)
Target: x86_64-apple-darwin20.2.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld" -demangle -lto_library /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/libLTO.dylib -dynamic -arch x86_64 -platform_version macos 11.0.0 11.1 -syslibroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -o cmTC_fe95f -search_paths_first -headerpad_max_install_names -v CMakeFiles/cmTC_fe95f.dir/CMakeCXXCompilerABI.cpp.o -lc++ -lSystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/lib/darwin/libclang_rt.osx.a
@(#)PROGRAM:ld  PROJECT:ld64-609.8
BUILD 15:07:46 Dec 18 2020
configured to support archs: armv6 armv7 armv7s arm64 arm64e arm64_32 i386 x86_64 x86_64h armv6m armv7k armv7m armv7em
Library search paths:
	/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib
Framework search paths:
	/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks/



Parsed CXX implicit include dir info from above output: rv=done
  found start of include info
  found start of implicit include info
    add: [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1]
    add: [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include]
    add: [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include]
    add: [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include]
  end of search list found
  collapse include dir [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1] ==> [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1]
  collapse include dir [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include] ==> [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include]
  collapse include dir [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include] ==> [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include]
  collapse include dir [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include] ==> [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include]
  implicit include dirs: [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1;/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include;/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include;/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include]


Parsed CXX implicit link information from above output:
  link line regex: [^( *|.*[/\])(ld|CMAKE_LINK_STARTFILE-NOTFOUND|([^/\]+-)?ld|collect2)[^/\]*( |$)]
  ignore line: [Change Dir: /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeTmp]
  ignore line: []
  ignore line: [Run Build Command(s):/usr/bin/make cmTC_fe95f/fast && /Applications/Xcode.app/Contents/Developer/usr/bin/make  -f CMakeFiles/cmTC_fe95f.dir/build.make CMakeFiles/cmTC_fe95f.dir/build]
  ignore line: [Building CXX object CMakeFiles/cmTC_fe95f.dir/CMakeCXXCompilerABI.cpp.o]
  ignore line: [/usr/bin/clang++   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk   -v -Wl -v -o CMakeFiles/cmTC_fe95f.dir/CMakeCXXCompilerABI.cpp.o -c /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/CMakeCXXCompilerABI.cpp]
  ignore line: [Apple clang version 12.0.0 (clang-1200.0.32.29)]
  ignore line: [Target: x86_64-apple-darwin20.2.0]
  ignore line: [Thread model: posix]
  ignore line: [InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin]
  ignore line: [clang: warning: -Wl -v: 'linker' input unused [-Wunused-command-line-argument]]
  ignore line: [ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include/c++/v1"]
  ignore line: [ "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang" -cc1 -triple x86_64-apple-macosx11.0.0 -Wdeprecated-objc-isa-usage -Werror=deprecated-objc-isa-usage -Werror=implicit-function-declaration -emit-obj -mrelax-all -disable-free -disable-llvm-verifier -discard-value-names -main-file-name CMakeCXXCompilerABI.cpp -mrelocation-model pic -pic-level 2 -mthread-model posix -mframe-pointer=all -fno-strict-return -masm-verbose -munwind-tables -target-sdk-version=11.1 -target-cpu penryn -dwarf-column-info -debugger-tuning=lldb -target-linker-version 609.8 -v -resource-dir /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -stdlib=libc++ -internal-isystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1 -internal-isystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/local/include -internal-isystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include -internal-externc-isystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include -internal-externc-isystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include -Wno-reorder-init-list -Wno-implicit-int-float-conversion -Wno-c99-designator -Wno-final-dtor-non-final-class -Wno-extra-semi-stmt -Wno-misleading-indentation -Wno-quoted-include-in-framework-header -Wno-implicit-fallthrough -Wno-enum-enum-conversion -Wno-enum-float-conversion -fdeprecated-macro -fdebug-compilation-dir /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeTmp -ferror-limit 19 -fmessage-length 0 -stack-protector 1 -fstack-check -mdarwin-stkchk-strong-link -fblocks -fencode-extended-block-signature -fregister-global-dtors-with-atexit -fgnuc-version=4.2.1 -fobjc-runtime=macosx-11.0.0 -fcxx-exceptions -fexceptions -fmax-type-align=16 -fdiagnostics-show-option -o CMakeFiles/cmTC_fe95f.dir/CMakeCXXCompilerABI.cpp.o -x c++ /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/CMakeCXXCompilerABI.cpp]
  ignore line: [clang -cc1 version 12.0.0 (clang-1200.0.32.29) default target x86_64-apple-darwin20.2.0]
  ignore line: [ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/local/include"]
  ignore line: [ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/Library/Frameworks"]
  ignore line: [#include "..." search starts here:]
  ignore line: [#include <...> search starts here:]
  ignore line: [ /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1]
  ignore line: [ /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include]
  ignore line: [ /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include]
  ignore line: [ /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include]
  ignore line: [ /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks (framework directory)]
  ignore line: [End of search list.]
  ignore line: [Linking CXX executable cmTC_fe95f]
  ignore line: [/usr/local/Cellar/cmake/3.19.4/bin/cmake -E cmake_link_script CMakeFiles/cmTC_fe95f.dir/link.txt --verbose=1]
  ignore line: [/usr/bin/clang++  -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -Wl -search_paths_first -Wl -headerpad_max_install_names  -v -Wl -v CMakeFiles/cmTC_fe95f.dir/CMakeCXXCompilerABI.cpp.o -o cmTC_fe95f ]
  ignore line: [Apple clang version 12.0.0 (clang-1200.0.32.29)]
  ignore line: [Target: x86_64-apple-darwin20.2.0]
  ignore line: [Thread model: posix]
  ignore line: [InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin]
  link line: [ "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld" -demangle -lto_library /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/libLTO.dylib -dynamic -arch x86_64 -platform_version macos 11.0.0 11.1 -syslibroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -o cmTC_fe95f -search_paths_first -headerpad_max_install_names -v CMakeFiles/cmTC_fe95f.dir/CMakeCXXCompilerABI.cpp.o -lc++ -lSystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/lib/darwin/libclang_rt.osx.a]
    arg [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld] ==> ignore
    arg [-demangle] ==> ignore
    arg [-lto_library] ==> ignore, skip following value
    arg [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/libLTO.dylib] ==> skip value of -lto_library
    arg [-dynamic] ==> ignore
    arg [-arch] ==> ignore
    arg [x86_64] ==> ignore
    arg [-platform_version] ==> ignore
    arg [macos] ==> ignore
    arg [11.0.0] ==> ignore
    arg [11.1] ==> ignore
    arg [-syslibroot] ==> ignore
    arg [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk] ==> ignore
    arg [-o] ==> ignore
    arg [cmTC_fe95f] ==> ignore
    arg [-search_paths_first] ==> ignore
    arg [-headerpad_max_install_names] ==> ignore
    arg [-v] ==> ignore
    arg [CMakeFiles/cmTC_fe95f.dir/CMakeCXXCompilerABI.cpp.o] ==> ignore
    arg [-lc++] ==> lib [c++]
    arg [-lSystem] ==> lib [System]
    arg [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/lib/darwin/libclang_rt.osx.a] ==> lib [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/lib/darwin/libclang_rt.osx.a]
  Library search paths: [;/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib]
  Framework search paths: [;/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks/]
  remove lib [System]
  remove lib [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/lib/darwin/libclang_rt.osx.a]
  collapse library dir [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib] ==> [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib]
  collapse framework dir [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks/] ==> [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks]
  implicit libs: [c++]
  implicit dirs: [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib]
  implicit fwks: [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks]


Compiling the C compiler identification source file "CMakeCCompilerId.c" succeeded.
Compiler: /usr/bin/gcc 
Build flags: 
Id flags:  

The output was:
0


Compilation of the C compiler identification source "CMakeCCompilerId.c" produced "a.out"

The C compiler identification is AppleClang, found in "/Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/3.19.4/CompilerIdC/a.out"

Detecting C compiler ABI info compiled with the following output:
Change Dir: /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeTmp

Run Build Command(s):/usr/bin/make cmTC_59dcc/fast && /Applications/Xcode.app/Contents/Developer/usr/bin/make  -f CMakeFiles/cmTC_59dcc.dir/build.make CMakeFiles/cmTC_59dcc.dir/build
Building C object CMakeFiles/cmTC_59dcc.dir/CMakeCCompilerABI.c.o
/usr/bin/gcc   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk   -v -Wl,-v -o CMakeFiles/cmTC_59dcc.dir/CMakeCCompilerABI.c.o -c /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/CMakeCCompilerABI.c
Apple clang version 12.0.0 (clang-1200.0.32.29)
Target: x86_64-apple-darwin20.2.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
clang: warning: -Wl,-v: 'linker' input unused [-Wunused-command-line-argument]
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang" -cc1 -triple x86_64-apple-macosx11.0.0 -Wdeprecated-objc-isa-usage -Werror=deprecated-objc-isa-usage -Werror=implicit-function-declaration -emit-obj -mrelax-all -disable-free -disable-llvm-verifier -discard-value-names -main-file-name CMakeCCompilerABI.c -mrelocation-model pic -pic-level 2 -mthread-model posix -mframe-pointer=all -fno-strict-return -masm-verbose -munwind-tables -target-sdk-version=11.1 -target-cpu penryn -dwarf-column-info -debugger-tuning=lldb -target-linker-version 609.8 -v -resource-dir /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -internal-isystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/local/include -internal-isystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include -internal-externc-isystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include -internal-externc-isystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include -Wno-reorder-init-list -Wno-implicit-int-float-conversion -Wno-c99-designator -Wno-final-dtor-non-final-class -Wno-extra-semi-stmt -Wno-misleading-indentation -Wno-quoted-include-in-framework-header -Wno-implicit-fallthrough -Wno-enum-enum-conversion -Wno-enum-float-conversion -fdebug-compilation-dir /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeTmp -ferror-limit 19 -fmessage-length 0 -stack-protector 1 -fstack-check -mdarwin-stkchk-strong-link -fblocks -fencode-extended-block-signature -fregister-global-dtors-with-atexit -fgnuc-version=4.2.1 -fobjc-runtime=macosx-11.0.0 -fmax-type-align=16 -fdiagnostics-show-option -o CMakeFiles/cmTC_59dcc.dir/CMakeCCompilerABI.c.o -x c /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/CMakeCCompilerABI.c
clang -cc1 version 12.0.0 (clang-1200.0.32.29) default target x86_64-apple-darwin20.2.0
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/local/include"
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/Library/Frameworks"
#include "..." search starts here:
#include <...> search starts here:
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks (framework directory)
End of search list.
Linking C executable cmTC_59dcc
/usr/local/Cellar/cmake/3.19.4/bin/cmake -E cmake_link_script CMakeFiles/cmTC_59dcc.dir/link.txt --verbose=1
/usr/bin/gcc  -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  -v -Wl,-v CMakeFiles/cmTC_59dcc.dir/CMakeCCompilerABI.c.o -o cmTC_59dcc 
Apple clang version 12.0.0 (clang-1200.0.32.29)
Target: x86_64-apple-darwin20.2.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld" -demangle -lto_library /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/libLTO.dylib -dynamic -arch x86_64 -platform_version macos 11.0.0 11.1 -syslibroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -o cmTC_59dcc -search_paths_first -headerpad_max_install_names -v CMakeFiles/cmTC_59dcc.dir/CMakeCCompilerABI.c.o -lSystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/lib/darwin/libclang_rt.osx.a
@(#)PROGRAM:ld  PROJECT:ld64-609.8
BUILD 15:07:46 Dec 18 2020
configured to support archs: armv6 armv7 armv7s arm64 arm64e arm64_32 i386 x86_64 x86_64h armv6m armv7k armv7m armv7em
Library search paths:
	/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib
Framework search paths:
	/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks/



Parsed C implicit include dir info from above output: rv=done
  found start of include info
  found start of implicit include info
    add: [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include]
    add: [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include]
    add: [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include]
  end of search list found
  collapse include dir [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include] ==> [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include]
  collapse include dir [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include] ==> [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include]
  collapse include dir [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include] ==> [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include]
  implicit include dirs: [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include;/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include;/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include]


Parsed C implicit link information from above output:
  link line regex: [^( *|.*[/\])(ld|CMAKE_LINK_STARTFILE-NOTFOUND|([^/\]+-)?ld|collect2)[^/\]*( |$)]
  ignore line: [Change Dir: /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeTmp]
  ignore line: []
  ignore line: [Run Build Command(s):/usr/bin/make cmTC_59dcc/fast && /Applications/Xcode.app/Contents/Developer/usr/bin/make  -f CMakeFiles/cmTC_59dcc.dir/build.make CMakeFiles/cmTC_59dcc.dir/build]
  ignore line: [Building C object CMakeFiles/cmTC_59dcc.dir/CMakeCCompilerABI.c.o]
  ignore line: [/usr/bin/gcc   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk   -v -Wl -v -o CMakeFiles/cmTC_59dcc.dir/CMakeCCompilerABI.c.o -c /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/CMakeCCompilerABI.c]
  ignore line: [Apple clang version 12.0.0 (clang-1200.0.32.29)]
  ignore line: [Target: x86_64-apple-darwin20.2.0]
  ignore line: [Thread model: posix]
  ignore line: [InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin]
  ignore line: [clang: warning: -Wl -v: 'linker' input unused [-Wunused-command-line-argument]]
  ignore line: [ "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang" -cc1 -triple x86_64-apple-macosx11.0.0 -Wdeprecated-objc-isa-usage -Werror=deprecated-objc-isa-usage -Werror=implicit-function-declaration -emit-obj -mrelax-all -disable-free -disable-llvm-verifier -discard-value-names -main-file-name CMakeCCompilerABI.c -mrelocation-model pic -pic-level 2 -mthread-model posix -mframe-pointer=all -fno-strict-return -masm-verbose -munwind-tables -target-sdk-version=11.1 -target-cpu penryn -dwarf-column-info -debugger-tuning=lldb -target-linker-version 609.8 -v -resource-dir /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -internal-isystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/local/include -internal-isystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include -internal-externc-isystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include -internal-externc-isystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include -Wno-reorder-init-list -Wno-implicit-int-float-conversion -Wno-c99-designator -Wno-final-dtor-non-final-class -Wno-extra-semi-stmt -Wno-misleading-indentation -Wno-quoted-include-in-framework-header -Wno-implicit-fallthrough -Wno-enum-enum-conversion -Wno-enum-float-conversion -fdebug-compilation-dir /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeTmp -ferror-limit 19 -fmessage-length 0 -stack-protector 1 -fstack-check -mdarwin-stkchk-strong-link -fblocks -fencode-extended-block-signature -fregister-global-dtors-with-atexit -fgnuc-version=4.2.1 -fobjc-runtime=macosx-11.0.0 -fmax-type-align=16 -fdiagnostics-show-option -o CMakeFiles/cmTC_59dcc.dir/CMakeCCompilerABI.c.o -x c /usr/local/Cellar/cmake/3.19.4/share/cmake/Modules/CMakeCCompilerABI.c]
  ignore line: [clang -cc1 version 12.0.0 (clang-1200.0.32.29) default target x86_64-apple-darwin20.2.0]
  ignore line: [ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/local/include"]
  ignore line: [ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/Library/Frameworks"]
  ignore line: [#include "..." search starts here:]
  ignore line: [#include <...> search starts here:]
  ignore line: [ /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include]
  ignore line: [ /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/include]
  ignore line: [ /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include]
  ignore line: [ /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks (framework directory)]
  ignore line: [End of search list.]
  ignore line: [Linking C executable cmTC_59dcc]
  ignore line: [/usr/local/Cellar/cmake/3.19.4/bin/cmake -E cmake_link_script CMakeFiles/cmTC_59dcc.dir/link.txt --verbose=1]
  ignore line: [/usr/bin/gcc  -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -Wl -search_paths_first -Wl -headerpad_max_install_names  -v -Wl -v CMakeFiles/cmTC_59dcc.dir/CMakeCCompilerABI.c.o -o cmTC_59dcc ]
  ignore line: [Apple clang version 12.0.0 (clang-1200.0.32.29)]
  ignore line: [Target: x86_64-apple-darwin20.2.0]
  ignore line: [Thread model: posix]
  ignore line: [InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin]
  link line: [ "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld" -demangle -lto_library /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/libLTO.dylib -dynamic -arch x86_64 -platform_version macos 11.0.0 11.1 -syslibroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -o cmTC_59dcc -search_paths_first -headerpad_max_install_names -v CMakeFiles/cmTC_59dcc.dir/CMakeCCompilerABI.c.o -lSystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/lib/darwin/libclang_rt.osx.a]
    arg [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld] ==> ignore
    arg [-demangle] ==> ignore
    arg [-lto_library] ==> ignore, skip following value
    arg [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/libLTO.dylib] ==> skip value of -lto_library
    arg [-dynamic] ==> ignore
    arg [-arch] ==> ignore
    arg [x86_64] ==> ignore
    arg [-platform_version] ==> ignore
    arg [macos] ==> ignore
    arg [11.0.0] ==> ignore
    arg [11.1] ==> ignore
    arg [-syslibroot] ==> ignore
    arg [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk] ==> ignore
    arg [-o] ==> ignore
    arg [cmTC_59dcc] ==> ignore
    arg [-search_paths_first] ==> ignore
    arg [-headerpad_max_install_names] ==> ignore
    arg [-v] ==> ignore
    arg [CMakeFiles/cmTC_59dcc.dir/CMakeCCompilerABI.c.o] ==> ignore
    arg [-lSystem] ==> lib [System]
    arg [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/lib/darwin/libclang_rt.osx.a] ==> lib [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/lib/darwin/libclang_rt.osx.a]
  Library search paths: [;/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib]
  Framework search paths: [;/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks/]
  remove lib [System]
  remove lib [/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/lib/darwin/libclang_rt.osx.a]
  collapse library dir [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib] ==> [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib]
  collapse framework dir [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks/] ==> [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks]
  implicit libs: []
  implicit dirs: [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/usr/lib]
  implicit fwks: [/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk/System/Library/Frameworks]


Determining if the include file pthread.h exists passed with the following output:
Change Dir: /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeTmp

Run Build Command(s):/usr/bin/make cmTC_762be/fast && /Applications/Xcode.app/Contents/Developer/usr/bin/make  -f CMakeFiles/cmTC_762be.dir/build.make CMakeFiles/cmTC_762be.dir/build
Building CXX object CMakeFiles/cmTC_762be.dir/CheckIncludeFile.cxx.o
/usr/bin/clang++   -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -std=gnu++17 -o CMakeFiles/cmTC_762be.dir/CheckIncludeFile.cxx.o -c /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeTmp/CheckIncludeFile.cxx
Linking CXX executable cmTC_762be
/usr/local/Cellar/cmake/3.19.4/bin/cmake -E cmake_link_script CMakeFiles/cmTC_762be.dir/link.txt --verbose=1
/usr/bin/clang++  -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names CMakeFiles/cmTC_762be.dir/CheckIncludeFile.cxx.o -o cmTC_762be 



Performing C++ SOURCE FILE Test CMAKE_HAVE_LIBC_PTHREAD succeeded with the following output:
Change Dir: /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeTmp

Run Build Command(s):/usr/bin/make cmTC_bcc4a/fast && /Applications/Xcode.app/Contents/Developer/usr/bin/make  -f CMakeFiles/cmTC_bcc4a.dir/build.make CMakeFiles/cmTC_bcc4a.dir/build
Building CXX object CMakeFiles/cmTC_bcc4a.dir/src.cxx.o
/usr/bin/clang++ -DCMAKE_HAVE_LIBC_PTHREAD  -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -std=gnu++17 -o CMakeFiles/cmTC_bcc4a.dir/src.cxx.o -c /Users/xxx/git_projekte/build-Lichtsteuerung-Desktop_Qt_5_15_2_clang_64bit-Debug/CMakeFiles/CMakeTmp/src.cxx
Linking CXX executable cmTC_bcc4a
/usr/local/Cellar/cmake/3.19.4/bin/cmake -E cmake_link_script CMakeFiles/cmTC_bcc4a.dir/link.txt --verbose=1
/usr/bin/clang++  -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names CMakeFiles/cmTC_bcc4a.dir/src.cxx.o -o cmTC_bcc4a 


Source file was:
#include <pthread.h>

static void* test_func(void* data)
{
  return data;
}

int main(void)
{
  pthread_t thread;
  pthread_create(&thread, NULL, test_func, NULL);
  pthread_detach(thread);
  pthread_cancel(thread);
  pthread_join(thread, NULL);
  pthread_atfork(NULL, NULL, NULL);
  pthread_exit(NULL);

  return 0;
}

</pre>
</details>

But building only with 
```cmake
find_package(Boost COMPONENTS context thread REQUIRED)
target_link_libraries(lichtsteuerung Boost::boost Boost::context Boost::thread)
```
works, but then I get the following linker errors:

```
Undefined symbols for architecture x86_64:
  "_jump_fcontext", referenced from:
      boost::context::fiber::resume() && in boostloopprogramcontextswitcher.cpp.o
      void* boost::context::detail::create_fiber2<boost::context::detail::fiber_record<boost::context::fiber, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::xxxs2::detail::pull_xxx<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)>, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::xxxs2::detail::pull_xxx<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)>(boost::context::preallocated, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&, boost::context::detail::fiber_record<boost::context::fiber, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::xxxs2::detail::pull_xxx<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)>&&) in boostloopprogramcontextswitcher.cpp.o
      void boost::context::detail::fiber_entry<boost::context::detail::fiber_record<boost::context::fiber, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::xxxs2::detail::pull_xxx<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)> >(boost::context::detail::transfer_t) in boostloopprogramcontextswitcher.cpp.o
  "_make_fcontext", referenced from:
      void* boost::context::detail::create_fiber2<boost::context::detail::fiber_record<boost::context::fiber, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::xxxs2::detail::pull_xxx<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)>, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::xxxs2::detail::pull_xxx<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)>(boost::context::preallocated, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&, boost::context::detail::fiber_record<boost::context::fiber, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::xxxs2::detail::pull_xxx<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)>&&) in boostloopprogramcontextswitcher.cpp.o
  "_ontop_fcontext", referenced from:
      boost::context::fiber::~fiber() in boostloopprogramcontextswitcher.cpp.o
      void boost::context::detail::fiber_entry<boost::context::detail::fiber_record<boost::context::fiber, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::xxxs2::detail::pull_xxx<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)> >(boost::context::detail::transfer_t) in boostloopprogramcontextswitcher.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [src/lichtsteuerung] Error 1
make[2]: *** [src/CMakeFiles/lichtsteuerung.dir/all] Error 2
make[1]: *** [src/CMakeFiles/lichtsteuerung.dir/rule] Error 2
make: *** [lichtsteuerung] Error 2
02:01:30: The process "/usr/local/Cellar/cmake/3.19.4/bin/cmake" exited with code 2.
Error while building/deploying project lichtsteuerung (kit: Desktop Qt 5.15.2 clang 64bit)
When executing step "Build"
```

When I install boost with [brew](https://brew.sh/) and do the following:
```cmake
target_link_libraries(lichtsteuerung "-L/usr/local/lib -lboost_coroutine -lboost_context-mt")
```
then the project builds and links

> @autoantwort, boost-corotine2 is header only, it depends on boost-context, could you try the following usage instead?
> 
> ```
> find_package(Boost REQUIRED COMPONENTS context)
> target_link_libraries(main PRIVATE ${Boost_LIBRARIES})
> target_include_directories(main PUBLIC ${Boost_INCLUDE_DIRS})
> ```

Then I get the following linker errors:
```
Undefined symbols for architecture x86_64:
  "boost::thread_detail::enter_once_region(boost::once_flag&)", referenced from:
      void boost::call_once<void (*)(rlimit*), rlimit*>(boost::once_flag&, void (*)(rlimit*), rlimit*) in libboost_context.a(stack_traits.o)
      void boost::call_once<void (*)(unsigned long*), unsigned long*>(boost::once_flag&, void (*)(unsigned long*), unsigned long*) in libboost_context.a(stack_traits.o)
  "boost::thread_detail::commit_once_region(boost::once_flag&)", referenced from:
      void boost::call_once<void (*)(rlimit*), rlimit*>(boost::once_flag&, void (*)(rlimit*), rlimit*) in libboost_context.a(stack_traits.o)
      void boost::call_once<void (*)(unsigned long*), unsigned long*>(boost::once_flag&, void (*)(unsigned long*), unsigned long*) in libboost_context.a(stack_traits.o)
  "boost::thread_detail::rollback_once_region(boost::once_flag&)", referenced from:
      void boost::call_once<void (*)(rlimit*), rlimit*>(boost::once_flag&, void (*)(rlimit*), rlimit*) in libboost_context.a(stack_traits.o)
      void boost::call_once<void (*)(unsigned long*), unsigned long*>(boost::once_flag&, void (*)(unsigned long*), unsigned long*) in libboost_context.a(stack_traits.o)
  "_jump_fcontext", referenced from:
      boost::context::fiber::resume() && in boostloopprogramcontextswitcher.cpp.o
      void* boost::context::detail::create_fiber2<boost::context::detail::fiber_record<boost::context::fiber, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::coroutines2::detail::pull_coroutine<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)>, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::coroutines2::detail::pull_coroutine<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)>(boost::context::preallocated, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&, boost::context::detail::fiber_record<boost::context::fiber, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::coroutines2::detail::pull_coroutine<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)>&&) in boostloopprogramcontextswitcher.cpp.o
      void boost::context::detail::fiber_entry<boost::context::detail::fiber_record<boost::context::fiber, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::coroutines2::detail::pull_coroutine<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)> >(boost::context::detail::transfer_t) in boostloopprogramcontextswitcher.cpp.o
  "_make_fcontext", referenced from:
      void* boost::context::detail::create_fiber2<boost::context::detail::fiber_record<boost::context::fiber, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::coroutines2::detail::pull_coroutine<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)>, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::coroutines2::detail::pull_coroutine<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)>(boost::context::preallocated, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&, boost::context::detail::fiber_record<boost::context::fiber, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::coroutines2::detail::pull_coroutine<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)>&&) in boostloopprogramcontextswitcher.cpp.o
  "_ontop_fcontext", referenced from:
      boost::context::fiber::~fiber() in boostloopprogramcontextswitcher.cpp.o
      void boost::context::detail::fiber_entry<boost::context::detail::fiber_record<boost::context::fiber, boost::context::basic_fixedsize_stack<boost::context::stack_traits>, boost::coroutines2::detail::pull_coroutine<void>::control_block::control_block<boost::context::basic_fixedsize_stack<boost::context::stack_traits>, Modules::BoostLoopProgramContextSwitcher::start()::$_0>(boost::context::preallocated, boost::context::basic_fixedsize_stack<boost::context::stack_traits>&&, Modules::BoostLoopProgramContextSwitcher::start()::$_0&&)::'lambda'(boost::context::fiber&&)> >(boost::context::detail::transfer_t) in boostloopprogramcontextswitcher.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [src/lichtsteuerung] Error 1
make[2]: *** [src/CMakeFiles/lichtsteuerung.dir/all] Error 2
make[1]: *** [src/CMakeFiles/lichtsteuerung.dir/rule] Error 2
make: *** [lichtsteuerung] Error 2
14:33:43: The process "/usr/local/Cellar/cmake/3.19.4/bin/cmake" exited with code 2.
Error while building/deploying project lichtsteuerung (kit: Desktop Qt 5.15.2 clang 64bit)
When executing step "Build"
```
Same as before + additional linker errors for `Boost::thread`

--- a/ports/boost-context/b2-options.cmake.in
+++ b/ports/boost-context/b2-options.cmake.in
@@ -5,3 +5,10 @@ if(@VCPKG_TARGET_IS_WINDOWS@)
         binary-format=pe
     )
 endif()
+
+if(@VCPKG_TARGET_IS_OSX@)
+    list(APPEND B2_OPTIONS
+        abi=sysv
+        binary-format=mach-o
+    )
+endif()

--- a/ports/boost-context/vcpkg.json
+++ b/ports/boost-context/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "boost-context",
   "version-string": "1.75.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Boost context module",
   "homepage": "https://github.com/boostorg/context",
   "supports": "!uwp & !emscripten",

--- a/versions/b-/boost-context.json
+++ b/versions/b-/boost-context.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "fbf3476b81167645a5f053adfdb7f2bf9a4f4c44",
+      "version-string": "1.75.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "7f3c7f2174e6d43195a6468272d27bf68b12b56b",
       "version-string": "1.75.0",
       "port-version": 1

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -494,7 +494,7 @@
     },
     "boost-context": {
       "baseline": "1.75.0",
-      "port-version": 1
+      "port-version": 2
     },
     "boost-contract": {
       "baseline": "1.75.0",

